test/SlowTests/Cluster/ClusterTransactionTests.cs

```diff
@@ -109,6 +109,7 @@ private string RandomString(int length)
         [InlineData(5)]
         public async Task CanPreformSeveralClusterTransactions(int numberOfNodes)
         {
+            NoTimeouts();
```

Comment:
Can we do that? I think we are reusing the servers, no?

Reply:
so? are we relying on timeouts in the tests?

Reply:
I can set no timeouts only for the server in hand, and restore the timeout once the test is over